lib/PSR/ContextBuilder.hs

```diff
@@ -49,46 +48,50 @@ import PlutusLedgerApi.V3.EvaluationContext qualified as V3
 -- Q: Is there a better way to represent this?
 
 -- Context0 is immediately derivable from the transactions.
-data Context0 where
+data Context0 era where
     Context0 ::
         { ctxPrevChainPoint :: C.ChainPoint
         , ctxShelleyBasedEra :: C.ShelleyBasedEra era
```

Comment:
We might be able to remove this now, since the outer Block contains it, it can be passed to any function that needs it.